include/sparrow/arrow_interface/arrow_array_schema_proxy.hpp

```diff
@@ -790,6 +790,11 @@ namespace sparrow
          */
         [[nodiscard]] SPARROW_API arrow_proxy view() const;
 
+        /**
+         * Check whether the proxy is a view.
+         */
+        [[nodiscard]] SPARROW_API constexpr bool is_view() const noexcept;
```

Comment:
constexpr functions must be defined when declared. And I'm not sure this function can be constexpr actually.